ESokha/05/Lesson_5_task_5_part_1.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html> 
+<HTML lang="ru"> 
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>Lesson_5_task_5_part_1</TITLE>
+		<style type="text/css">
+			.main {
+			display: flex;
+			margin: 0 auto;
+			margin-bottom: 0.2em;
+					
+			}
+			.mainn {
+			display: flex;
+			margin: 0 auto;
+			background-color: rgb(180, 181, 181);
+			
+			}
+			
+			.navigation {
+			width: 15em;
```

Comment:
If you use flex, then all size for block will be set by flex property

Reply:
sorry, i correcct this mistake and it will pushed tomorrow